Wikipedia/Code/WMFViewController.m

```diff
@@ -25,7 +25,7 @@ @implementation WMFViewController
 
 - (void)setup {
     self.theme = [WMFTheme standard];
-    self.navigationBar = [[WMFNavigationBar alloc] initWithFrame:CGRectZero];
+    self.navigationBar = [[WMFNavigationBar alloc] initWithFrame:CGRectMake(0, 0, 320, 44)];
```

Comment:
Rather than have to manually set an arbitrary frame in two places, it might be simpler to have the standard `init` call `initWithFrame` with the default value

Reply:
Ah, can do!

Reply:
@joewalsh done!